doc/developer/lua.rst

```diff
@@ -0,0 +1,59 @@
+LUA
```

Comment:
please put `.. _lua` on the first line of the file followed by a blank line

Reply:
fixed